tasks/install.yml

```diff
@@ -13,6 +13,7 @@
     - python-pip
     - libsasl2-dev
     - libldap2-dev
+    - libmysqlclient-dev
```

Comment:
why add mysql?

Reply:
When we need to build dashboards off a mysql database (like the ILRI case) we need a python mysql library for Superset to be able to deal with that database